src/main/java/com/twitter/whiskey/net/SpdyConstants.java

```diff
@@ -13,6 +13,7 @@
     static final int PRIORITY_LEVELS = 8;
     static final int SPDY_SESSION_STREAM_ID = 0;
     static final int DEFAULT_INITIAL_WINDOW_SIZE = 65536;
+    static final int DEFAULT_MAX_CONCURRENT_STREAMS = 100;
```

Comment:
yes, definitely - SPDY 3.1 spec says there is no maximum by default so we can just saturate the data type.